libfreerdp/codec/xcrush.c

```diff
@@ -821,7 +834,7 @@ static int xcrush_decompress_l1(XCRUSH_CONTEXT* xcrush, const BYTE* pSrcData, UI
 				    (&Literals[OutputLength] > pSrcEnd))
 					return -1009;
 
-				xcrush_copy_bytes(HistoryPtr, Literals, OutputLength);
```

Comment:
Revert this, your assumption that source and destination are different might not hold.

Reply:
No problem to me